src/lib/fixer.ts

```diff
@@ -111,14 +111,14 @@ export class _ApexGuruFixGenerator extends FixGenerator {
     
         const action = new vscode.CodeAction(messages.fixer.fixWithApexGuruSuggestions, vscode.CodeActionKind.QuickFix);
         action.diagnostics = [this.diagnostic];
-
-        const edit = new vscode.WorkspaceEdit();
         const range = this.diagnostic.range;  // Assuming the range is the location of the existing code in the document
         const diagnosticStartLine = new vscode.Position(range.start.line, range.start.character);
-        edit.insert(document.uri, diagnosticStartLine, suggestedCode + '\n');
```

Comment:
If I do this, there is no way to get the telemetry on how many times users clicked on the "include apex guru suggestions" quick-fix. I had to to make this a command and add it to the action and I can report telemetry on that command.